server/chat-plugins/chat-monitor.js

```diff
@@ -301,6 +300,42 @@ let nicknamefilter = function (name, user, forStatus) {
 
 	return name;
 };
+/** @type {StatusFilter} */
+let statusfilter = function (status, user) {
+	let lcStatus = status.replace(/\u039d/g, 'N').toLowerCase().replace(/[\u200b\u007F\u00AD]/g, '').replace(/\u03bf/g, 'o').replace(/\u043e/g, 'o').replace(/\u0430/g, 'a').replace(/\u0435/g, 'e').replace(/\u039d/g, 'e');
+	// Remove false positives.
+	lcStatus = lcStatus.replace('herapist', '').replace('grape', '').replace('scrape', '');
+	// Check for blatant staff impersonation attempts. Ideally this could be completely generated from Config.grouplist
+	// for better support for side servers, but not all ranks are staff ranks or should necessarily be filted.
+	if (/(global|room|upper|senior)?\s*(staff|admin|adminstrator|leader|owner|mod|moderator|driver|voice|operator|sysop|creator)/gi.test(lcStatus)) {
```

Comment:
``s/adminstrator/administrator``

Since there is "creator", please add "founder" as well, which is common in side servers.

Reply:
But do we really want to allow people to typo administrator? ;)